lms/djangoapps/verify_student/tests/test_views.py

```diff
@@ -1888,5 +1888,5 @@ def _get_url(self, course_key, checkpoint):
                        kwargs={
                            "course_id": unicode(course_key),
                            "checkpoint_name": checkpoint,
-                           "location": unicode(self.reverification_location)
+                           "usage_id": unicode(self.reverification_location)
```

Comment:
For my own edification, could you please explain the difference between `location` and `usage_id`? I don't understand why it's been changed throughout.